apps/laboratory/src/components/AppKitInfo.tsx

```diff
@@ -55,6 +59,42 @@ export function AppKitInfo({ caipAddress, address, chainId, clientId }: AppKitIn
               <Text data-testid="w3m-chain-id">{clientId}</Text>
             </Box>
           )}
+
+          <Box>
+            <Heading size="xs" textTransform="uppercase" pb="2">
+              Account Type
+            </Heading>
+            <Text data-testid="w3m-account-type">
+              {accountType === 'eoa' ? 'EOA' : 'Smart Account'}
+            </Text>
+          </Box>
+
+          {email && (
+            <Box>
+              <Heading size="xs" textTransform="uppercase" pb="2">
+                Email
+              </Heading>
+              <Text data-testid="w3m-email">{email}</Text>
+            </Box>
+          )}
+
+          {username && (
+            <Box>
+              <Heading size="xs" textTransform="uppercase" pb="2">
+                Username
+              </Heading>
+              <Text>{username}</Text>
+            </Box>
+          )}
+
+          <Box>
+            <Heading size="xs" textTransform="uppercase" pb="2">
+              Smart Account Status
+            </Heading>
+            <Text data-testid="w3m-sa-account-status">
+              {isSmartAccountDeployed ? 'Deployed' : 'Not Deployed'}
+            </Text>
+          </Box>
```

Comment:
maybe nit but can we make a component for this?  like `InfoRow` or something

Reply:
good idea